wiki/BanchoBot/en.md

```diff
@@ -1,28 +1,154 @@
-| --------- | ---------------------- | ----------- | ------------- | ---------------- |
-| `HELP`    | None                   | Shows the list of available commands                                                    | `!help`                               | This list |
-| `ROLL`    | `[number]`             | Randomly gets a value up to the selected number (default is 100).                       | `!roll 1000`                          | "Nathanael rolls 789 point(s)" |
-| `STATS`   | `<user_name>`          | Displays the stats of the user depending on the mode they're playing                    | `!stats peppy`                        | "Stats for [peppy](https://osu.ppy.sh/u/2): <br> Score:    412,018,739 (#94718) <br> Plays:    7073 (lv65) <br> Accuracy: 87.31%" |
-| `WHERE`   | `<user_name>`          | Shows the current location of the user.                                                 | `!where Kyubey`                       | "Kyubey is in Russia" |
-| `FAQ`     | `[lang:]<entry>/list`  | Displays the message of the entry. Use `list` as entry to show all available entries.   | `!faq ping`                           | "Pong!" |
-| `REPORT`  | `<user_name> <reason>` | Sends a report to the chat moderators. Replace username spaces to underscores `_`       | `!report S_o_h spamming in #japanese` | "Chat moderators have been alerted. Thanks for your help." |
-| `REQUEST` | None                   | Receive a random beatmap that a user requests modding assistance.                       | `!request`                            | "[HoneyWorks - Tokyo Summer Session feat. CHiCO](https://osu.ppy.sh/s/426252) by MrSergio" |
-<!-- Note: The !search command is not included because it is no longer working. -->
+*For a list of in-game client commands, see: [Chat Console](/wiki/Chat_Console#commands-list)*
+
+BanchoBot commands start with `!` immediately followed by the command. These commands work in-game and in external IRC clients and can be used in multiplayer chat channels and in BanchoBot's PM (private message). If a normal user sends a BanchoBot command in a public chat channel, other players will not see it, but the response will be displayed in a private message from BanchoBot.
+
+You can also use the `/bb` in-game client command to automatically open a BanchoBot tab and send the command immediately.
+
+*Notice: The `SEARCH` command is not included as it is not working.*
+
+*Note: The user argument is case-insensitive, as usernames are case-insensitive.*
+
+### HELP
+
+```
+!HELP
+```
+
+Help shows the list of available commands.
+
+#### Usage Examples
+
+*Notice: The `user` parameter for the `REPORT` command is missing.*
+
+```
+pippi: !help
+BanchoBot: Standard Commands (!COMMAND or /msg BanchoBot COMMAND):
+BanchoBot: SEARCH <query>|next|prev
+BanchoBot: WHERE <user>
+BanchoBot: STATS <user>
+BanchoBot: FAQ <item>|list
+BanchoBot: REPORT <reason> - call an admin
```

Comment:
Did you intentionally reword some of BanchoBot's responses?

Reply:
Not intentionally, I hand typed those and while I was reading BanchoBot's responses, I kept adding those words in unconsciously.